src/System.Management.Automation/security/SecureStringHelper.cs

```diff
@@ -389,7 +390,9 @@ internal static SecureString Decrypt(string input, byte[] key, byte[] IV)
 #nullable enable
         internal static unsafe SecureString FromPlainTextString(string plainTextString)
         {
-            if (plainTextString.Length == 0)
+            Debug.Assert(plainTextString is not null);
+
+            if (string.IsNullOrEmpty(plainTextString))
```

Comment:
If we is IsNullOrEmpty we can remove Debug.Assert

Reply:
I think the current configuration makes sense; we *shouldn't* accept `null`, but if it's given to us in the product, we'll do the right thing.

Reply:
The alternative is that we should simply return `null` when given `null`

Reply:
I've removed the null check and added documentation.